zoidberg/zoidberg.py

```diff
@@ -450,7 +454,17 @@ def _write_par_metric(self, maps, nslice, ypar):
                 yperiodic=yperiodic,
             )
 
-            par_metric, _, _ = get_metric(par_grid, self.field)
+            par_metric, _, _, par_BJg = get_metric(par_grid, self.field)
+
+            if self.BJg is not None and self.BJg.shape[0] > 4:
+                mymax = np.max(np.abs(par_BJg / self.BJg - 1)[2:-2], axis=(0, 2))
+                if np.max(mymax) > 1e-6:
+                    print(
+                        "FluxError",
+                        np.max(np.abs(par_BJg / self.BJg - 1)[2:-2], axis=(0, 2)),
```

Comment:
Is this not just `mymax`?
```suggestion
                        mymax
```

Also, should this instead be a warning or an error?